extension/src/popup.html

```diff
@@ -5,7 +5,7 @@
     <title>ROSE Wallet Extension</title>
   </head>
   <body>
-    <div id="root" style="min-width: 360px; min-height: 600px"></div>
+    <div id="root" style="min-width: 375px; min-height: 600px"></div>
```

Comment:
Aligning popup width to EN tab labels seems naive. Anyway, if we need more space can we go with current/old extension 450 instead? More langs will look Ok. Maybe we should use "Accounts" label too to save some space. It's quite clear that accounts listed under profile belongs to user.

Reply:
Preview of german:
![Screenshot from 2024-04-25 17-33-27](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/ef8f590b-d5e9-442c-92e7-8ef2474a4e8f)

Reply:
I'd like a wider extension too, and we can update that very easily, but we'll probably want to support 375px iphones either way